src/features/courses/course-controller.ts

```diff
@@ -1150,6 +1165,7 @@ class CourseController {
                 formData = gradeInstance.currentProblemState;
                 sourceFilePath = gradeInstance.webworkQuestionPath;
                 problemSeed = gradeInstance.randomSeed;
+                calculatedRendererParameters.outputformat = OutputFormat.ASSESS;
```

Comment:
Why not add this to the getCalculatedRendererParams?
Maybe add an option for isAssessment that returns ASSESS

Also this look like too generic of a way of putting this here, if we ever have a non null workbook (like viewing previous submissions, I'm guessing that this is in one of the get grades and I would assume this would break the workbook veiwer (but I could be wrong)

Either way moving the logic into the calculate renderer params should make it more clear

Reply:
this should be moved - agreed - but adding reliance on topicType to the existing logic was a complication where the concern was breaking existing functionality... will handle this in cleanup.

Reply:
I'm fine with that

However my concern is more so with adding extra functionality that will change the existing

We've been kicking a lot of stuff downstream and the result is that either:
1. We will forget about it
2. Higher priority tasks will build up between where we will build this into a deeper level of code that is harder to touch
3. Bugs / problems will come of it before we get there (i.e. if we do this outside of the calculate and need it in multiple places but do it differently)